### PR TITLE
feat: add manage tax components component (#197)

### PR DESCRIPTION
**Pull request by Verdiane**
## Description
Add manage tax components component

## Related issues and discussion
   #197 

## Screenshots, if any

taxComponents

Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by Verdiane** [comment]
Hello @abhaychawla , Thanks for reviewing my PRs. I have done the required changes.
Look forward to working with you

File: src/app/products/manage-tax-components/manage-tax-components.component.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <tr mat-row *matRowDef="let row; columns: displayedColumns;" class="select-row"></tr>
+  
+      </table>
+  
+      <mat-paginator [pageSizeOptions]="[10, 25, 50, 100]" showFirstLastButtons></mat-paginator>
+  
+    </div>
+  
+  </div>
+
```
[the remark this answers is not in the document]
Comment (abhaychawla) [reply src/app/products/manage-tax-components/manage-tax-components.component.html]: No changes were made here.

File: src/app/products/manage-tax-components/manage-tax-components.component.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+table {
+    width: 100%;
+  
+    .select-row:hover {
+      cursor: pointer;
+    }
+  }
+
```
[the remark this answers is not in the document]
Comment (abhaychawla) [reply src/app/products/manage-tax-components/manage-tax-components.component.scss]: No changes were made here.

**Comment by Verdiane** [comment]
Hello @abhaychawla , I have made the required changes on those files. Thanks for the review

**Pull request merged**